resources_pages/install_ds_stack_mac.md

```diff
@@ -205,34 +234,43 @@ For more information about these matters see
 https://www.gnu.org/licenses/.
 ```
 
-Note: Although it is possible to install R through conda, we highly recommend not doing so. In case you have already installed R using conda you can remove it by executing `conda uninstall r-base`.
+> Note: Although it is possible to install R through conda, we highly recommend not doing so. In case you have already installed R using conda you can remove it by executing `conda uninstall r-base`.
 
 ### XQuartz
 
 Some R packages rely on the dependency XQuartz which no longer ships with the Mac OS, thus we need to install it separately. Download it from here: [https://www.xquartz.org/](https://www.xquartz.org/) and follow the installation instructions.
 
 ### RStudio
-Chose and download the Mac version of RStudio from [https://www.rstudio.com/products/rstudio/download/#download](https://www.rstudio.com/products/rstudio/download/#download). Open the file and follow the installer instructions.
 
-To see if you were successful, try opening RStudio by clicking on its icon (from Finder, Applications or Launchpad). It should open and looks something like this picture below:
+Download the macOS Desktop version of RStudio Preview from [https://rstudio.com/products/rstudio/download/preview/](https://rstudio.com/products/rstudio/download/preview/). Open the file and follow the installer instructions.
+
+To see if you were successful, try opening RStudio by clicking on its icon (from Finder, Applications or Launchpad). It should open and look something like this picture below:
 
 ![](/resources_pages/imgs/RStudio.png)
 
-### IR kernel
 
-Open RStudio and type the following commands into the Console panel:
+### Essential R packages
+
+Next, install the key R packages needed for the start of MDS program,
+by typing the following into the R terminal in RStudio:
 
 ```
-install.packages(c('IRkernel', 'tidyverse', 'blogdown', 'xaringan', 'renv'))
+install.packages(c('tidyverse', 'blogdown', 'xaringan', 'renv'))
 ```
 
-Next, open **terminal** and type the following:
+> Note: we will use many more packages than those listed above across the MDS program, however we will manage these using the `renv` package manager (which you will learn about in DSCI 521: Platforms for Data Science).
+
+### IR kernel
+
+The `IRkernel` package is needed to make R work in Jupyter notebooks. To enable this kernel in the notebooks, install it and run the setup via the following two commands:
+Open RStudio and type the following commands into the Console panel:
 
 ```
-R -e "IRkernel::installspec()"
+install.packages('IRkernel')
+IRkernel::installspec()
```

Comment:
This needs to be done in terminal on a Mac, not RStudio. From the IRkernel install instructions:

2/2) Making the kernel available to Jupyter

The kernel spec can be installed for the current user with the following line from R:
```
IRkernel::installspec()
```
On OS X, be sure to execute this in R started from the Terminal, not the R App!
(This is because the R app doesn’t honor $PATH changes in ~/.bash_profile)

Source: https://irkernel.github.io/installation/

Reply:
My bad, thanks for noticing!

Reply:
I notice because I have missed this before and it took a while to figure out what was going wrong. We remember the things that hurt 😅